Appraisals

```diff
@@ -95,7 +95,7 @@ RAILS_VERSIONS = %w(
   4.2.5.1
   4.2.5.2
   4.2.6
-  5.0.0.beta3
+  5.0.0.1
```

Comment:
> OK, don't take it as an offence, but what will this knowledge give you?

What if the behavior changes because of the CVE patch? Without testing the previous version you can't know it.



> Would you add tests for private methods defined in a model/class?

Nope ;)